.gitignore

```diff
@@ -36,6 +36,8 @@ _ReSharper*/
 *.userprefs
 *.swp
 *.DotSettings
+nuget.package/lib/
```

Comment:
You don't use `lib` there anymore, so it can go from .gitignore. And if you specify the LICENSE.md in the .nuspec, then you can also let go the App_Readme folder (here and on the disk).

Reply:
updated accordingly